src/main/java/org/generationcp/middleware/dao/NameDAO.java

```diff
@@ -184,6 +183,19 @@ public Name getByGIDAndNval(final Integer gid, final String nval) {
 		return new ArrayList<>();
 	}
 
+	public Name getPreferredNameByGid(final Integer gid) {
```

Comment:
If we reused `getGermplasmDtoById`, this is not needed

Reply:
Done